action/kafkaProduce.py

```diff
@@ -45,5 +54,19 @@ def validateParams(params):
 
     if len(missingParams) > 0:
         return (False, "You must supply all of the following parameters: {}".format(', '.join(missingParams)))
+    elif 'base64DecodeValue' in params and params['base64DecodeValue'] == True and not isBase64(params['value']):
+        return (False, "value parameter is not base64 encoded")
+    elif 'base64DecodeKey' in params and params['base64DecodeKey'] == True and not isBase64(params['key']):
+        return (False, "key parameter is not base64 encoded")
     else:
         return (True, "Params all check out.")
+
+def isBase64(s):
+    print "In isBase64"
+    valid = True
+    try:
+        valid = (base64.b64encode(base64.b64decode(s)) == s)
```

Comment:
If you use the `.decode("base64")` method, decoding values that result from trying to decode a non-base64 value are an empty string. It seems like it'd be must each to check string length instead of what you are doing here.